src/context/meta-tags-context.php

```diff
@@ -655,6 +674,40 @@ public function generate_main_image_id() {
 		}
 	}
 
+	/**
+	 * Gets the main image ID for REST requests.
+	 *
+	 * @return int|null The main image ID.
+	 */
+	protected function get_main_image_id_for_rest_request() {
```

Comment:
Why `protected` and not `private` like you did in your URL method?
Also, I think we used to order these in a class, public before protected & private. Not sure how well we actually did that or if we still value that to be honest.

Reply:
My bad, it was meant to be private 🙂